backend/corpora/common/utils/corpora_constants.py

```diff
@@ -80,11 +80,9 @@ class CorporaConstants(object):
     ]
 
     REQUIRED_SIMPLE_METADATA_FIELDS = [
-        "version",
```

Comment:
do we want to replace this with `schema_version`?

Reply:
Missed that - thank you!